lib/ret/hub.ex

```diff
@@ -72,7 +72,8 @@ defmodule Ret.Hub do
   def changeset_for_new_spawned_object_type(%Hub{} = hub, object_type)
       when object_type in 0..31 do
     # spawned_object_types is a bitmask of the seen object types
-    new_spawned_object_types = hub.spawned_object_types ||| 1 <<< object_type
+    <<new_spawned_object_types::integer-signed-32>> =
+      <<hub.spawned_object_types ||| 1 <<< object_type::integer-signed-32>>
```

Comment:
Took me a while to realize what you were doing here. Do we really want to store overflowed values in the DB, instead of just making the field a `bigint`? 
I would add a comment here at the very least.

Reply:
i'll take a pass at converting the column type -- i was a bit nervous about it since it's a a change to the telemetry table but shouldn't be too bad